doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -131,6 +131,13 @@ Matter
     To enable diagnostic logs support, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS` Kconfig option to ``y``.
     To learn more about the diagnostic logs module see :ref:`ug_matter_configuration_diagnostic_logs`.
 
+  * Crash logs support for the diagnostic logs provider.
+    This feature requires the retained RAM memory partition to store a crash data after its occurrence.
+    To enable crash logs support, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_CRASH_LOGS` Kconfig option to ``y`` and build a sample with an extra devicetree overlay file: :file:`../common/src/{board}_retention.overlay`.
```

Comment:
Not sure if it's worth repeating the instruction in the release notes. Just link to the doc?